Form/Handler.php

```diff
@@ -206,7 +207,9 @@ protected function sendMails(
                 $type->getCustomerToMailAddress($form->getData()),
                 $type->getCustomerFromMailAddress($form->getData()),
                 true,
-                $type->getCustomerReplyToMailAddress($form->getData())
+                $type->getCustomerReplyToMailAddress($form->getData()),
+                $this->attachments,
```

Comment:
$type->getCustomerSendAttachments($form->getData()) ? $this->attachments : array()